Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -1526,8 +1539,10 @@ public static string Repr(CodeContext/*!*/ context, object? o)
                 }
             }
 
-            static object? CallPrepare(CodeContext/*!*/ context, object meta, string name, PythonTuple bases, PythonDictionary? keywords, PythonDictionary dict) {
-                object? classdict = dict;
+            static object? CallPrepare(CodeContext/*!*/ context, object? meta, string name, PythonTuple bases, PythonDictionary? keywords, PythonDictionary dict) {
```

Comment:
Is the return type still nullable?

Reply:
No, it is not indeed. But frankly the whole `CallPrepare` is slated to be reworked in the last step. I'm going to change the nullability now though, because you never know when there will be time to finish it.